# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../keytouch-editor/keytouch-editor-3.1.3-glibc28.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Fix compiling with glibc >= 2.8

Reference:

http://projects.archlinux.org/svntogit/packages.git/tree/keytouch-editor/trunk/keytouch-editor-3.1.3-glibc28.patch

diff -Naur keytouch-editor-3.1.3-orig/src/Makefile.am keytouch-editor-3.1.3/src/Makefile.am
--- keytouch-editor-3.1.3-orig/src/Makefile.am	2008-06-22 22:47:34.000000000 -0400
+++ keytouch-editor-3.1.3/src/Makefile.am	2008-06-22 22:49:52.000000000 -0400
@@ -9,7 +9,7 @@
 
 bin_PROGRAMS = keytouch-editor-bin
 
-AM_CFLAGS=@CFLAGS@ -Wall
+AM_CFLAGS=@CFLAGS@ -Wall -D_GNU_SOURCE
 
 keytouch_editor_bin_SOURCES = \
 	main.c keytouch-editor.h \
